docs/run/error-mitigation-explanation.ipynb

```diff
@@ -0,0 +1,284 @@
+    "The image below depicts digital gate folding on the left, and the extrapolation procedure on the right.\n",
+    "\n",
+    "![Depiction of ZNE](/images/run/error-mitigation-explanation/zne.png)\n",
+    "\n",
+    "ZNE can be enabled by setting `zne_mitigation` to `True` in version 2 of the [Qiskit Runtime resilience options](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ResilienceOptionsV2).\n",
+    "The Qiskit Runtime options for ZNE are described [here](https://docs.quantum.ibm.com/api/qiskit-ibm-runtime/qiskit_ibm_runtime.options.ZneOptions). The following options are notable:\n",
+    "\n",
+    "- `noise_factors`: The noise factors to use for noise amplification.\n",
+    "- `extrapolator`: The functional form to use for the extrapolation.\n",
+    "\n",
+    "The following code cell shows how enable ZNE and set these options for the estimator. None of these options are required to be set explicitly."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "06de4431",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "estimator = Estimator(backend=backend)\n",
+    "estimator.options.resilience.zne_mitigation = True\n",
+    "estimator.options.resilience.zne.noise_factors = (1, 3, 5)\n",
+    "estimator.options.resilience.zne.extrapolator = \"exponential\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "fcc40cb7-45b7-4248-84c4-aa7f26ba50ff",
+   "metadata": {},
+   "source": [
+    "## Probabilistic error cancellation (PEC)\n",
+    "\n",
+    "Probabilistic error cancellation (PEC) is a technique for mitigating errors in estimating expectation values of observables. Unlike ZNE, it returns an unbiased estimate of the expectation value. However, it generally incurs a greater overhead.\n",
+    "\n",
+    "In PEC, the effect of an ideal target circuit is expressed as a linear combination of noisy circuits that are actually implementable in practice:\n",
+    "\n",
+    "$$\n",
+    "\\mathcal{O}_{\\text{ideal}} = \\sum_{i} \\eta_i \\mathcal{O}_{noisy, i}\n",
+    "$$\n",
+    "\n",
+    "The output of the ideal circuit can then be reproduced by executing different noisy circuit instances drawn from a random ensemble defined by the linear combination. If the coefficients $\\eta_i$ form a probability distribution, they can be used directly as the probabilities of the ensemble. In practice, some of the coefficients are negative, so they form a quasiprobability distribution instead. They can still be used to define a random ensemble, but there is a sampling overhead related to the negativity of the quasiprobability distribution, which is characterized by the quantity\n",
```

Comment:
```suggestion
    "The output of the ideal circuit can then be reproduced by executing different noisy circuit instances drawn from a random ensemble defined by the linear combination. If the coefficients $\\eta_i$ form a probability distribution, they can be used directly as the probabilities of the ensemble. In practice, some of the coefficients are negative, so they form a quasiprobability distribution instead. They can still be used to define a random ensemble, but there is a sampling overhead related to the negativity of the quasi-probability distribution, which is characterized by the quantity\n",
```

Reply:
What's the reason for this suggestion?

Reply:
I knew you'd ask about this 🤣 The consensus at IBM Quantum is to use `quasi-probability distribution` as the standard. https://github.com/Qiskit/qiskit-ibm-runtime/pull/924